### PR TITLE
Update data-driven-forms

### PR DESCRIPTION
**Pull request by rvsia**
# Pull request motivation(s)

Expand crawling for another html elements (table, code, pre, ...)

https://github.com/data-driven-forms/react-forms/pull/261

### CODE REVIEW

File: configs/data-driven-forms.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "lvl3": ".DocSearch-content h4",
     "lvl4": ".DocSearch-content h5",
     "lvl5": ".DocSearch-content h6",
-    "text": ".DocSearch-content p, .DocSearch-content li"
+    "text": ".DocSearch-content p, .DocSearch-content li, .DocSearch-content pre, .DocSearch-content .DocTable, .DocSearch-content .DocCode, .DocSearch-content .DocRawComponent"
```
Comment (s-pace) [inline configs/data-driven-forms.json]: These selectors target code element or element that are too big. We do not recommend to index these kind of elements https://community.algolia.com/docsearch/faq.html#can-you-index-code-samples

Comment (rvsia) [reply configs/data-driven-forms.json]: Fixed!

**Comment by Hyperkid123** [comment]
@rvsia thanks

File: configs/data-driven-forms.json (hunk 1 above)
Comment (s-pace) [reply configs/data-driven-forms.json]: Great thanks

**Pull request merged**